deploy/charts/kubernetes-cluster-prep/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+name: conjur-cluster-prep
+description: A Helm chart for Kubernetes
+
```

Comment:
Please add:
```
icon: https://www.cyberark.com/wp-content/uploads/2015/12/cybr-aim.jpg
```
after description line.

Reply:
sure, just curious, where does this appear?

Reply:
https://stackoverflow.com/questions/58154658/where-is-a-helm-icon-url-used